app/components/Poll/Poll.css

```diff
@@ -63,61 +130,40 @@
   }
 }
 
-.pollHeader a {
-  font-weight: 500;
-  color: var(--lego-font-color);
-  line-height: 1.3;
-}
-
-.voteButton {
-  width: 100%;
-  margin: 1px !important;
+.resultsHiddenInfo {
+  font-style: italic;
```

Comment:
I think I prefer to not use any italic font style as we don't really use that anywhere else on the webapp (except a few anomalies). Can I suggest just a secondary font color?